frontend/src/services/notification/notification.service.ts

```diff
@@ -0,0 +1,27 @@
+import { BasicToastrOptions, toastr, ToastrEmitter } from 'react-redux-toastr';
+
+class Notification {
+  #instance: ToastrEmitter;
+
+  constructor() {
+    this.#instance = toastr;
+  }
+
+  public success(title: string, message: string, options?: BasicToastrOptions): void {
```

Comment:
Public is redundant

Reply:
it would be better to always indicate this
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-member-accessibility.md